new-log-viewer/src/components/StatusBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import Button from "@mui/joy/Button";
+import Sheet from "@mui/joy/Sheet";
+import Typography from "@mui/joy/Typography";
+
+
+interface StatusBarProps {
+    handleCopyLinkButtonClick: () => void;
+    logEventNum: number | null;
+    numEvents: number | null;
+}
+
+export const StatusBar: React.FC<StatusBarProps> = ({handleCopyLinkButtonClick, logEventNum, numEvents}) => {
+    return (
+        <Sheet className={"status-bar"}>
+            <Typography
+                className={"status-message"}
+                level={"body-sm"}
+            >
+                Status message
+            </Typography>
+            <Button
+                className={"status-button"}
+                size={"sm"}
+                onClick={handleCopyLinkButtonClick}
```

Comment:
Shall we copy the function definition to this source file to avoid passing the callback as a prop to the component?